Assets/MixedRealityToolkit-SDK/Features/Input/FocusProvider.cs

```diff
@@ -432,6 +437,23 @@ public bool RegisterPointer(IMixedRealityPointer pointer)
             return true;
         }
 
+        private void RegisterPointers(IMixedRealityInputSource inputSource)
```

Comment:
Can we make these RegisterPointer(s) calls more generic and put them in the InputSystem?
Seeing this duplicated / replicated around the project.

Reply:
That's a much larger breaking change than a simple bug fix.

Reply:
Fair enough, make a task for it later :D

Reply:
Much appreciated. I do agree we can refine this class a bit more and possibly make it a full proper manager. For now I want to get the rest of the input devices working first.